gapis/api/vulkan/api/pipeline.api

```diff
@@ -664,12 +675,23 @@ cmd void vkDestroyPipeline(
 // Shader modules //
 ////////////////////
 
+@internal class UsedDescriptor {
+  u32 Set
+  u32 Binding
+  u32 DescriptorCount
+}
+
+@internal class UsedDescriptors {
```

Comment:
Do we really need to wrap this map in a type, or we can just use the map reference directly?

Also, it would be clearer if the map is named "EntryPointsAndUsedDescriptors". A lot of following places use this class in a way like `Descriptors.Descriptors.xxx`, which is not quite clear.

Reply:
Long story short: Yes because we attach this as an extra